Firebase/Auth/Source/RPCs/FIRAuthBackend.h

```diff
@@ -464,6 +466,7 @@ typedef void (^FIRVerifyClientResponseCallback)
 - (void)sendVerificationCode:(FIRSendVerificationCodeRequest *)request
```

Comment:
Please include this method in `#if TARGET_OS_IOS` as it is used by Phone Number Auth only.